docs/sdk/index.mdx

```diff
@@ -0,0 +1,247 @@
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Updating user attributes</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Getting user information</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Getting evaluation details</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Listening to evaluation updates</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Background mode</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+        <td><UncheckSVG/></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+
+
+### Server-side SDK
+
+Server-side SDKs are designed for multi-user systems and intended for use in trusted environments, like corporate networks or web servers. They operate within server-architected applications running on your infrastructure or trusted cloud-based infrastructure. These locations are not directly accessible by end-users. The restricted access of server-based applications allows server-side SDKs to safely receive flag data and rulesets without the need to filter out sensitive information. This setup enables server-side SDKs to receive flag data including rules, and segments safely without filtering out sensitive information.
+
+Supported SDKs for server-side:
+
+<div className="row" style={{maxWidth: '500px'}}>
+
+  <div className="col--3 text--center">
+    <a href="/sdk#server-side-sdk" className="brand-link">
+      <i className="golang-icon brand-icon"></i>
+      <span>Go</span>
+    </a>
+  </div>
+
+  <div className="col--3 text--center">
+    <a href="/sdk#server-side-sdk" className="brand-link">
+      <i className="nodejs-icon brand-icon"></i>
+      <span>Node.js</span>
+    </a>
+  </div>
+
+</div>
+
+#### Server available features
+
+The table below illustrates all features available in each one of the available Server SDKs.
+
+<div className="sdk-table">
+  <table>
+    <thead>
+      <tr>
+        <th>Feature</th>
+        <th>Go</th>
+        <th>Node Js</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Evaluating user</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Variation types</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Reporting custom events</td>
+        <td><CheckSVG/></td>
+        <td><CheckSVG/></td>
+      </tr>
+      <tr>
+        <td>Destroy (same as Flushing events in the client SDK)</td>
```

Comment:
I think it is easier for the engineer to understand.

```suggestion
        <td>Flushing events</td>
```

Reply:
Done